docs/user/disconnected-cluster.md

```diff
@@ -1,184 +1,91 @@
+oc image mirror docker.io/redis:7.4.1 ${LOCAL_REGISTRY}/redis:7.4.1
 ```
 
-look for
-
-```yaml
-  - mirrors:
-    - <registry-hostname>:5000
-    source: quay.io
-```
+The mirroring instructions apply for partially disconnected env, where from your hub you can reach the internet.
+For fully disconnected env, you will need to mirror on connected machine to disk/usb and bring the mirrored content to hub. Then mirror to local registry.
 
-if not present, add it using
+Example:
 
 ```shell
-oc edit imagecontentsourcepolicy image-policy
+#on machine connected to internet
+oc image mirror quay.io/keycloak/keycloak:25.0.1 file://myrepository/keycloak/keycloak:25.0.1
+#on disconnected env
+oc image mirror file://myrepository/keycloak/keycloak:25.0.1 ${LOCAL_REGISTRY}/keycloak/keycloak:25.0.1
 ```
 
-## Mirror quay.io images
+## Configure image repository mirroring
 
-### see list of images to use
+Create `ImageTagMirrorSet` with the following content
 
 ```shell
-helm show values oci://quay.io/flightctl/charts/flightctl --version=0.4.0-rc3
-```
-
-### mirror flightctl images
-
-```shell
-oc image mirror -a combined-secret.json quay.io/flightctl/flightctl-worker:0.4.0-rc3  <registry-hostname>:5000/flightctl/flightctl-worker:0.4.0-rc3
-
-oc image mirror -a combined-secret.json quay.io/flightctl/flightctl-periodic:0.4.0-rc3  <registry-hostname>:5000/flightctl/flightctl-periodic:0.4.0-rc3
-
-oc image mirror -a combined-secret.json quay.io/flightctl/flightctl-ui:0.4.0-rc3  <registry-hostname>:5000/flightctl/flightctl-ui:0.4.0-rc3
-
-oc image mirror -a combined-secret.json quay.io/flightctl/flightctl-api:0.4.0-rc3  <registry-hostname>:5000/flightctl/flightctl-api:0.4.0-rc3
-```
-
-### mirror `keycloak`, `redis` and `postgresql` (please note: format of `keycloak` in `values.yaml` is different from others)
-
-```shell
-oc image mirror -a combined-secret.json quay.io/keycloak/keycloak:25.0.1 <registry-hostname>:5000/keycloak/keycloak:25.0.1
-
-oc image mirror -a combined-secret.json quay.io/sclorg/redis-7-c9s:latest <registry-hostname>:5000/sclorg/redis-7-c9s
-```
-
-> 📌 Note: PostgreSQL Version Upgrade will occur in `0.5.0`, so `postgresql-12-c8s` will not be required
-
-```shell
-oc image mirror -a combined-secret.json quay.io/sclorg/postgresql-12-c8s:latest <registry-hostname>:5000/sclorg/postgresql-12-c8s
-
-oc image mirror -a combined-secret.json quay.io/sclorg/postgresql-16-c9s:latest <registry-hostname>:5000/sclorg/postgresql-16-c9s
-```
-
-### `sha256`
-
-Look for (usually the last line)
-
-```kubernetes helm
-sha256:b33dedfeb245199a03122505d35d9c75ae01d77ffe39e0950ed98b0d02ecdf64 <registry-hostname>:5000/flightctl/flightctl-kv:latest
-```
-
-and add it to `values.yaml` ([See Example](#an-example-of-valuesyaml)). Please note
-
-- `@ha256` must be placed at the end of image name
-- `sha256`'s value must be placed under `tag`
+oc apply -f - <<EOF
+apiVersion: config.openshift.io/v1
+kind: ImageTagMirrorSet
```

Comment:
I'm not sure at all that this method of redirect by tag is even supported by Openshift...  for sure ImageDigestMirrorSet is supported.

Reply:
This is based on the official docs https://docs.redhat.com/en/documentation/openshift_container_platform/4.13/html/images/image-configuration#images-configuration-registry-mirror_image-configuration

```
For new clusters, you can use IDMS, ITMS, and ICSP CRs objects as desired. However, using IDMS and ITMS is recommended.
```

Reply:
For the future releases, we will want to add proper digest support, which will allow us to document IDMS too. That way user can pick their preferred way.